src/app/Processor/ProcessorDetailPage/ProcessorDetailPage.tsx

```diff
@@ -83,21 +112,87 @@ const ProcessorDetailPage = (): JSX.Element => {
       console.error(processorError);
       goToInstance();
     }
-  }, [bridgeError, processorError, goToHome, goToInstance]);
+    if (updateProcessorError) {
+      console.error(updateProcessorError);
```

Comment:
I think you should manage the "processor name already existing" scenario when the user is editing a processor. Similarly to [what was done here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/blob/main/src/app/Processor/CreateProcessorPage/CreateProcessorPage.tsx#L74) during creation.

This is an error caused directly by the user, so we should take care of it in this phase.

Reply:
Ok, I'll do